src/helpers/tracking.ts

```diff
@@ -0,0 +1,29 @@
+import Amplitude from "amplitude-js";
+
+const instance = Amplitude.getInstance();
+instance.init(process.env.REACT_APP_AMPLITUDE_API_KEY!, undefined, {
+  trackingOptions: {
+    ip_address: false,
+  },
+});
+
+type Properties = {
+  [key: string]: any;
+};
+
+export const logEvent = (type: string, properties?: Properties) => {
+  console.log(
+    `[TRACKING]: ${type}: ${properties ? JSON.stringify(properties) : ""}`,
+  );
+
+  if (properties) {
+    // Override the IP so we don't collect it
+    // eslint-disable-next-line no-param-reassign
+    properties.ip = "";
+  }
+
+  // Track only production URL (exclude all dev URLs)
+  if (window.location.host === "accountviewer.stellar.org") {
```

Comment:
I wonder if there's a better way to know if we're in prod environment in case the URL changes. Are there any environment variables we could use eg. `process.env.NODE_ENV` ?

Reply:
I didn't find anything better. `process.env.NODE_ENV` is for dev and prod. For us, any developer URL (dev or PR previews) will be like prod, which we don't want to track. :woman_shrugging: